epic_capybara/cli/bara.py

```diff
@@ -171,12 +171,14 @@ def bara(files, match, unmatch, serve):
         x_bounds = (x_min - 0.05 * x_range, x_min + 1.05 * x_range)
         y_bounds = (- 0.05 * y_max, 1.05 * y_max)
         # Set y range for histograms
-        fig.x_range = Range1d(
-            *x_bounds,
-            bounds=x_bounds)
-        fig.y_range = Range1d(
-            *y_bounds,
-            bounds=y_bounds)
+        if np.all(np.isfinite(x_bounds)):
+            fig.x_range = Range1d(
+                *x_bounds,
+                bounds=x_bounds)
+        if np.all(np.isfinite(y_bounds)):
+            fig.y_range = Range1d(
+                *y_bounds,
+                bounds=y_bounds)
```

Comment:
To help diagnose the underlying issue, mabye this?
```suggestion
                bounds=y_bounds)
        else:
            print(f"{branch_name}/{leaf_name} has infinite y bounds")
```